### ISSUE TITLE
When creating a new task the datepicker is not visible with Minimal theme.

### ISSUE THREAD

**Issue by shawnduggan** [jamiebrynes7/obsidian-todoist-plugin#106]
When creating a new task, on the task creation popup, the datepicker is not visible when using the Minimal theme. 

<img width="713" alt="Screen Shot 2021-06-02 at 8 21 08 AM" src="https://user-images.githubusercontent.com/3504807/120471861-9bf74780-c37b-11eb-9651-6a571cb0022f.png">

**Comment by jamiebrynes7** [comment]
Hey @shawnduggan, this is related to the styling that the Minimal theme applies to the `modal` class.

```css
.modal {
    border: none;
    background: var(--background-primary);
    border-radius: 10px;
    overflow: hidden;
}
```

The `overflow: hidden;` property prevents the calendar from spilling out the bounds of the modal. A quick fix would be to add a CSS snippet to override this:

```css
.modal {
  overflow: visible;
}
```

Its likely that this could also be fixed in this plugin's CSS, but this'll give you a workaround for the time being at least.

**Comment by jamiebrynes7** [comment]
This should be fixed in [v1.8.0](https://github.com/jamiebrynes7/obsidian-todoist-plugin/releases/tag/1.8.0)
